examples/integration-tutorials/routing-requests-based-on-message-content/guide/health_care_service.bal

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/log;
+
+listener http:Listener httpListener = new(9092);
+
+// Endpoint URL of the backend service
+http:Client healthcareEndpoint = new("http://localhost:9095/healthcare");
+
+// RESTful service
+@http:ServiceConfig {
+    basePath: "/hospitalMgtService"
+}
+service hospitalMgtService on httpListener {
+
+    @http:ResourceConfig {
+        methods: ["GET"],
+        path: "/getdoctor/{category}"
```

Comment:
Path could be just 'doctor' since from the HTTP method we know it's a GET. This is REST convention.